src/ImageDeleteModal.jsx

```diff
@@ -17,115 +17,96 @@ function sortTags(a, b) {
-            client.untagImage(image.isSystem, image.Id, tag.substring(0, i), tag.substring(i + 1, tag.length))
+            client.untagImage(imageWillDelete.isSystem, imageWillDelete.Id, tag.substring(0, i), tag.substring(i + 1, tag.length))
                     .then(() => {
                         if (tags.length > 0)
-                            this.handleRemoveImage(tags, all);
+                            handleRemoveImage(tags, all);
                     })
                     .catch(ex => {
                         const error = cockpit.format(_("Failed to remove image $0"), tag);
-                        this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                        onAddNotification({ type: 'danger', error, errorDetail: ex.message });
                     });
         }
-    }
-
-    render() {
-        const Dialogs = this.context;
-
-        const repoTags = Object.keys(this.state.tags).sort(sortTags);
-        const checkedTags = repoTags.filter(x => this.state.tags[x]);
-        return (
-            <Modal isOpen
-                   position="top" variant="medium"
-                   onClose={Dialogs.close}
-                   title={cockpit.format(_("Delete $0"), repoTags ? repoTags[0] : "")}
-                   footer={<>
-                       <Button id="btn-img-delete" variant="danger" isDisabled={checkedTags.length === 0}
-                               onClick={() => this.handleRemoveImage(checkedTags, checkedTags.length === repoTags.length)}>
-                           {_("Delete tagged images")}
-                       </Button>
-                       <Button variant="link" onClick={Dialogs.close}>{_("Cancel")}</Button>
-                   </>}
-            >
-                <Stack hasGutter>
-                    { repoTags.length > 1 && <StackItem>{_("Multiple tags exist for this image. Select the tagged images to delete.")}</StackItem> }
-                    <StackItem isFilled>
-                        { repoTags.map(x => {
-                            return (
-                                <Checkbox isChecked={checkedTags.indexOf(x) > -1}
-                                          id={"delete-" + x}
-                                          aria-label={x}
-                                          key={x}
-                                          label={x}
-                                          onChange={checked => this.onValueChanged(x, checked)} />
-                            );
-                        })}
-                    </StackItem>
-                </Stack>
-                { repoTags.length > 2 && <Button variant="link" onClick={this.pickAll}>{_("select all")}</Button> }
-            </Modal>
-        );
-    }
-}
+    };
+
+    return (
+        <Modal isOpen
+                 position="top" variant="medium"
+                 onClose={Dialogs.close}
+                 title={cockpit.format(_("Delete $0"), repoTags ? repoTags[0] : "")}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20221130-163743-c39fb479-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)